src/components/StopPointsEditor/Generic/GenericStopPointEditor.tsx

```diff
@@ -80,11 +91,29 @@ export const GenericStopPointEditor = ({
           />
         </div>
 
-        <DeleteButton
-          disabled={!canDelete}
-          onClick={() => setDeleteDialogOpen(true)}
-          title={formatMessage({ id: 'editorDeleteButtonText' })}
-        />
+        <div className={'buttons-group'}>
+          <DeleteButton
+            thin={true}
+            disabled={!canDelete}
+            onClick={() => setDeleteDialogOpen(true)}
+            title={formatMessage({ id: 'editorDeleteButtonText' })}
+          />
+
+          {isLocateButtonAvailable && (
+            <TertiaryButton
```

Comment:
This is an interesting design question. I feel like we are leaking sandbox feature details into the main code here.

Reply:
Agreed! I will try to move this into a component and plug in as a Sandbox feature under JourneyPatternStopPointMap folder. But there are a couple more places with similar problem but would be more cumbersome to avoid

Reply:
Although, this particular case starts to seem bit cumbersome too 😅